crates/cargo-remove/src/cargo/ops/cargo_remove/manifest.rs

```diff
@@ -374,15 +374,13 @@ impl LocalManifest {
     pub fn remove_from_table(&mut self, table_path: &[String], name: &str) -> CargoResult<()> {
         let parent_table = self.get_table_mut(table_path)?;
 
-        {
-            let dep = parent_table
-                .get_mut(name)
-                .filter(|t| !t.is_none())
-                .ok_or_else(|| non_existent_dependency_err(name, table_path.join(".")))?;
+        let dep = parent_table
+            .get_mut(name)
+            .filter(|t| !t.is_none())
+            .ok_or_else(|| non_existent_dependency_err(name, table_path.join(".")))?;
 
-            // remove the dependency
-            *dep = toml_edit::Item::None;
-        }
+        // remove the dependency
+        *dep = toml_edit::Item::None;
```

Comment:
> fix(remove): remove unnecessary block 

For future reference, this would be a `refactor` as it has no user-facing impact

Reply:
okay! I can just fix this super quick